dpgen/dispatcher/LSF.py

```diff
@@ -1,31 +1,36 @@
-import os,getpass,time
+import os
+import getpass
+import time
 from dpgen.dispatcher.Batch import Batch
 from dpgen.dispatcher.JobStatus import JobStatus
 
-def _default_item(resources, key, value) :
-    if key not in resources :
+
+def _default_item(resources, key, value):
+    if key not in resources:
         resources[key] = value
 
-class LSF(Batch) :
-    
+
+class LSF(Batch):
+
     def check_status(self):
         try:
             job_id = self._get_job_id()
         except:
             return JobStatus.terminated
-        if job_id == "" :
-            raise RuntimeError("job %s is has not been submitted" % self.remote_root)
+        if job_id == "":
+            raise RuntimeError(
+                "job %s is has not been submitted" % self.context.remote_root)
         ret, stdin, stdout, stderr\
-            = self.context.block_call ("bjobs " + job_id)
+            = self.context.block_call("bjobs " + job_id)
         err_str = stderr.read().decode('utf-8')
-        if ("Job <%s> is not found" % job_id) in err_str :
-            if self.check_finish_tag() :
+        if ("Job <%s> is not found" % job_id) in err_str:
+            if self.check_finish_tag():
                 return JobStatus.finished
-            else :
+            else:
                 return JobStatus.terminated
-        elif ret != 0 :
-            raise RuntimeError ("status command bjobs fails to execute. erro info: %s return code %d"
-                                    % (err_str, ret))
```

Comment:
you should not change the format of the code (even though you do no like it). 

Such changes make people difficult to locate the real change to the code

Reply:
Thanks for your advise. I just committed to restore the format of the code to show where I modified. Sorry for using autopep8 directly to format the code without asking in ahead.